docs/src/main/asciidoc/reactive-routes.adoc

```diff
@@ -193,10 +196,70 @@ The annotated method can accept arguments of the following types:
 * `io.vertx.reactivex.core.http.HttpServerRequest`
 * `io.vertx.reactivex.core.http.HttpServerResponse`
 
-If the annotated method returns `void` then it has to accept at least one argument. 
-If the annotated method does not return `void` then the arguments are optional.
+Furhtermore, it is possible to inject the `HttpServerRequest` parameters into a method parameter annotated with `@io.quarkus.vertx.web.Param`:
+
+[options="header",cols="1,1"]
+|===
+|Parameter Type   | Obtained via  
+//-------------
+|`java.lang.String`   |`routingContext.request().getParam()`   
+|`java.util.Optional<String>`   |`routingContext.request().getParam()`   
+|`java.util.List<String>`   |`routingContext.request().params().getAll()`
+|===
+
+.Request Parameter Example
+[source,java]
+----
+@Route
+String hello(@Param Optional<String> name) {
```

Comment:
Isn't a compilation flag required to read attribute names?

Reply:
You're right. We should improve the error message...

Reply:
FYI: https://github.com/quarkusio/quarkus/issues/4632#issuecomment-657407680